src/utils.ts

```diff
@@ -237,13 +237,17 @@ export function getPreimage(tx, inputLockingScriptASM: string, inputAmount: numb
 // Converts a number into a sign-magnitude representation of certain size as a string
 // Throws if the number cannot be accommodated
 // Often used to append numbers to OP_RETURN, which are read in contracts
-// TODO: handle bigint
 export function num2bin(n: number, dataLen: number): string {
+  return pack(n, dataLen);
```

Comment:
Put `pack()` body into num2bin and remove `pack()` since they are the same. Same for `unpack`.